src/test/java/org/opentripplanner/middleware/utils/DateTimeUtilsTest.java

```diff
@@ -46,6 +48,14 @@ private static Stream<Arguments> createDateFormatCases() {
         );
     }
 
+    @Test
+    void canGetNextTimeFrom() {
+        var zdt = ZonedDateTime.of(2024, 11, 11, 15, 34, 17, 0, DateTimeUtils.getOtpZoneId());
+        var timeOfDay = LocalTime.parse("05:15");
+        var zdtNext = DateTimeUtils.getNextTimeFrom(timeOfDay, zdt);
+        assertTrue(zdtNext.isAfter(zdt));
```

Comment:
Since you are using an absolute date `zdt`, can you assert on the date, hour, and minutes instead?